taxonomy-editor-frontend/src/pages/home/index.jsx

```diff
@@ -0,0 +1,63 @@
+import Button from '@mui/material/Button';
+import {Link as MuiLink} from '@mui/material';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { Link } from "react-router-dom";
+
+function Copyright() {

```

Comment:
If not resused, this could be part of the code, no need to have an extra component